README.md

```diff
@@ -100,6 +100,13 @@ HTTP client configuration:
 - `maxFreeSockets` - Maximum number of sockets to leave open in a free
   state. Only relevant if `keepAlive` is set to `true` (default: `256`)
 
+APM Agent Configuration via Kibana:
+
+- `remoteConfig` - Wheather or not the client should poll the APM Server
```

Comment:
The name of this config option is still up for debate. I probably like `configPolling` better 🤔

Reply:
Looks like we've narrowed the candidates to either `remoteConfig` or `enableRemoteConfig`

Reply:
Updated it to `centralConfig` based on https://github.com/elastic/apm/issues/76#issuecomment-509379996